src/config/logger.ts

```diff
@@ -105,6 +105,19 @@ const censorUrl = (url) => {
 	return url;
 };
 
+const headersSerializer = (headers) => {
+	if (!headers) {
+		return headers;
+	}
+
+	const ret = mapKeys({ ...headers }, (_, key) => key.toLowerCase());
```

Comment:
why are you recreating an object from the headers?  Headers should already be an object :P

Reply:
Immutability, they say ;)

Reply:
Ah wait a sec, lodash is immtable 🤦  I'll fix that, good catch @mboudreau